doc/howToUse.rst

```diff
@@ -3,30 +3,29 @@
 How to use eTraGo?
 ==================
 
-After you installed eTraGo you can run eTraGo via terminal with
-``$ cd../<your path>/eTrago/etrago/`` and ``python3 appl.py``.
-The program will execute following functions:
+After you installed eTraGo you would typically start optimization runs by
+executing the ‘appl.py’ wich is situated in 
+``./eTrago/etrago/`` (e.g by ``python3 appl.py``).
 
-.. code-block:: python
-
-   # execute etrago function
-    network = etrago(args)
-    # plots
-    # make a line loading plot
-    plot_line_loading(network)
-    # plot stacked sum of nominal power for each generator type and timestep
-    plot_stacked_gen(network, resolution="MW")
-    # plot to show extendable storages
-    storage_distribution(network)
+The ‘appl.py’ is used as a simple user interface. Here
+parameters, calculation methods and scenario settings are set in a python
+dictionary called 'args'. It is crucial to understand these parameters.
+For example some of them contradict the usage of others. 
+The documentation of all parameters being defined within the 'args' you
+find here: :meth:`etrago.appl.etrago`.
```

Comment:
I suggest: You find the documentation of all defined parameters from the 'args' here: :meth:'etrago.appl.etrago'.